SAPHana/srHook/SAPHanaSrMultiTarget.py

```diff
@@ -133,11 +142,13 @@ def srConnectionChanged(self, ParamDict, **kwargs):
             myInSync = ParamDict["is_in_sync"]
             myReason = ParamDict["reason"]
             mySite = ParamDict["siteName"]
-            myCMD = "sudo /usr/sbin/crm_attribute -n hana_{1}_gsh -v {0} -l reboot".format(srHookGen, mysid)
-            rc = os.system(myCMD)
-            myMSG = "CALLING CRM: <{0}> rc={1}".format(myCMD, rc)
-            self.tracer.info("{0}.{1}() {2}\n".format(self.__class__.__name__, method, myMSG))
-            self.tracer.info("{0}.{1}() Running srHookGeneration {2}, see attribute hana_{3}_gsh too\n".format(self.__class__.__name__, method, srHookGen, mysid))
+            # if self.cib_access != "all-off" and self.cib_access != "glob-off":
+            if self.cib_access == "all-on" or self.cib_access == "glob-on" or self.cib_access == "site-off":
```

Comment:
Looks good for me but could get a growing "or" list. Alternatively we could set specific variables for each cib_call at a central code point and then could ease the if statements here. So e.g. if the value is "all-on" then cibAccGsh=1, cinGlobSrHook=1 etc.And here we only ask attr-specific if (cibAccGsh). Just an idea.

Reply:
yes, nice idea. I will take this into account when we will do the next changes to the hook or when we 'fine tune' the documentation and therefore add some more values to 'cib_access' which need to handle.